ui/js/directives/treeherder/clonejobs.js

```diff
@@ -219,8 +222,11 @@ treeherder.directive('thCloneJobs', [
             return visible;
         };
 
-        var addJobBtnEls = function(jgObj, jobList) {
+        var renderJobBtnEls = function(jgObj) {
             var lastJobSelected = ThResultSetStore.getSelectedJob($rootScope.repoName);
+            if (!_.isEmpty(lastJobSelected.job)) {
+                lastJobSelected.el = $('.' + getJobMapKey(lastJobSelected.job));
+            }
```

Comment:
I *think* this will work, but it should be called after we've rerendered all the job rows, not every time we render them, perhaps with a comment like:
```js
// update the selected job element, in case it changed
```
I would put this at the end of the two functions which wind up calling this: generateJobElements and updateJobs.